desktop/core/ext-py3/boto-2.49.0/boto/gs/key.py

```diff
@@ -29,7 +29,7 @@
 from boto.s3.key import Key as S3Key
 from boto.s3.keyfile import KeyFile
 from boto.utils import compute_hash
-from boto.utils import get_utf8_value
+from boto.utils import get_utf8_value, get_utf8able_str
```

Comment:
Same here, can we figure out a way of doing this without directly modifying the files of an external library?

Reply:
Tried but was getting error, so added this missing method in the package instead.
Actually this was present in the boto repo's developing branch but it was not released ever.